lib/base-error.js

```diff
@@ -0,0 +1,32 @@
+var _ = require('underscore');
```

Comment:
I *think* the only thing that gets lost from underscore transitioning to lodash is being able to pass a scope as an argument to an iterator, which we can get around pretty easily.